### PR TITLE
PEP 662: Editable installs via virtual wheels

### PR DESCRIPTION
**Pull request by gaborbernat**
Signed-off-by: Bernát Gábor <gaborjbernat@gmail.com>

### CODE REVIEW

**Comment by brettcannon** [comment]
Would you mind word wrapping this? See https://www.python.org/dev/peps/pep-0012/#general for guidance.

**Comment by gaborbernat** [comment]
Uh, ok doing that makes a PITA to make amendments to the PEP because you have to reindent multiple lines manually after every edit. Ideally we should do that just before merging not?

**Comment by brettcannon** [comment]
If you're explicitly asking for the rest of the PEP to be reviewed before reflowing that's fine, but the reflow will need to happen before the PEP can be accepted.

**Comment by JelleZijlstra** [comment]
I don't think every edit needs to trigger a complete reflow. If you need to add a few words to the middle of a paragraph later, you can just have a shorter line in the middle and leave the rest of the paragraph alone. The thing to aim for is that every line stays within the length limit.

File: pep-9999.rst
Comment (pradyunsg) [inline pep-9999.rst:5]: Use the topic about this specific PEP draft, rather than PEP 660's topic?

```suggestion
Discussions-To: https://discuss.python.org/t/9071
```

File: pep-9999.rst
Comment (brettcannon) [inline pep-9999.rst:244]:
```suggestion
   startup of the interpreter. This is done by creating a ``pth`` file into the target
```

File: pep-9999.rst
Comment (brettcannon) [inline pep-9999.rst:246]:
```suggestion
   install -e <project_directory>`_ translates to. This solution is fast and
```

**Review by brettcannon — CHANGES_REQUESTED** [review]
LGTM overall, but I want to double-check that the `:pep:` directives actually turn into hyperlinks as I didn't think they did thanks to us not using sphinx.

**Review by brettcannon — CHANGES_REQUESTED** [review]
LGTM! Can you add me as the sponsor, take the next PEP number (662), and add me to `.github/CODEOWNERS`? I will then approve and merge this.

**Comment by gaborbernat** [comment]
Sure, I'll ask another day or two to do some clarifications that came up on the discussion topic 🙏 thanks!

**Comment by gaborbernat** [comment]
> LGTM! Can you add me as the sponsor, take the next PEP number (662), and add me to `.github/CODEOWNERS`? I will then approve and merge this.

All done now 👍 thanks!

**Comment by pradyunsg** [comment]
You'd wanna retitle the PR. :)

File: pep-0662.rst
Comment (layday) [inline pep-0662.rst:201]: Nit: should be "scheme", not "schema".

**Review by brettcannon — APPROVED** [review]

**Pull request merged**
